app/views/user_mailer/send_otp.html.erb

```diff
@@ -0,0 +1 @@
+Your OTP for <%= link_to 'Violet', root_url(subdomain: Subdomain.current.name) %> is: <%= @user.current_otp %>
```

Comment:
@ajaystha42 this should be `<%= link_to Subdomain.current.name, root_url(subdomain: Subdomain.current.name) %>`